style.md

```diff
@@ -545,6 +546,102 @@ const (
 // LogToStdout=0, LogToFile=1, LogToRemote=2
 ```
 
+### Use time
```

Comment:
Maybe `Use package time for time related types and operations`

Reply:
I would like to avoid making the titles too long. Is there something shorter we can put here?
How about, `Use package time to handle time` or  `Use "time" to handle time`?

Reply:
I like `Use "time" to handle time`